docs/ADR-67-configuring-wearables.md

```diff
@@ -0,0 +1,169 @@
+
+- The `id` property is the ID that the will have or has (if it's being updated) in the builder-server.
+- The `assetType` is used to identify the type of the asset, wether it's a wearable or a smart item.
+- The `name` is the name of the wearable, as it's seen in the Marketplace.
+- The `description` is the description of the wearable.
+- The `category` is the category of the wearable (eyewear, hat, etc).
+- The `rarity` is the rarity of the wearable (unique, common, etc).
+- The `thumbnail` is the path to the thumbnail of the wearable.
+- The `menuBarIcon` is the path to the image to use on the “experiences” menu, to represent this portable experience.
+- The `model` is the path to the model of the wearable. This implies that the wearable can have only one model for all representations.
+- The `bodyShape` is the body shape of the wearable. This can only have one value, either `male`, `female` or `both`, implying
+
+The current format has some downsides that we need to tackle:
+
+1. It doesn't have the possibility to define representations with different 3D models, that is, a female and a male representation, each one with its 3D model shaped to their body shape.
+2. Wearable tags can't be defined.
+3. It's not possible to define which categories get to be hidden or replaced by the wearable and, as it doesn't allow each representation to be detailed, it doesn't allow overriding hides and replaces for each of representation.
+4. The name of the file `asset.json` is confusing as assets already have a file with the same name on them.
+5. The `id` property is used to identify the item in the builder-server, this doesn't describe the wearable.
+
+These downsides make the current `asset.json` configuration file not suitable for describing wearables.
+
+## Proposed solution
+
+To solve the downsides described above, and to provide a configuration file that only describes wearables two changes are proposed:
+
+1. The configuration file (former `asset.json`) should be named `wearable.json`.
+2. A new format is defined for the `wearable.json` that describes everything about a wearable.
+3. All the information about the Builder and the platform is defined in another file in the [ADR-68](docs/ADR-68-importing-wearables.md).
+
+The format proposed is the following:
+
+```typescript
+type WearableAsset = {
+  /** Name of the wearable */
+  name: string;
+  /** Description of the wearable */
+  description?: string;
+  /** Rarity of the wearable */
+  rarity?: Rarity;
+  /** Category of the wearable */
+  category: WearableCategory;
+  /** Wearables to hide when equipping the wearable */
+  hides: WearableCategory[];
+  /** Wearables to replace when equipping the wearable */
+  replaces: WearableCategory[];
+  /** Tags to identify the wearable */
+  tags: string[];
+  /** Representations of the wearable */
+  representations: WearableRepresentation[];
+}
+
+type WearableRepresentation = {
+  /** Body shape of the representation */
+  bodyShape: BodyShape;
```

Comment:
This type is very similar to the `WearableRepresentation` that exists in the commons schemas repo: https://github.com/decentraland/common-schemas/blob/main/src/platform/wearables/representation.ts#L10-L16

Expect that the one in common schemas uses an array of urns instead of the enum `male`, `female`, `both`. I think it would be better to stick to the type we are already using even tho dealing with urns is more cumbersome for the sake of consistency. Also this design would be more future-proof if at some point more body shapes are supported.

Reply:
I've changed the whole schema to be identical to the wearable entitiy metadata.